dart/analysis_options.yaml

```diff
@@ -11,7 +11,11 @@ analyzer:
     # allow self-reference to deprecated members (we do this because otherwise we have
     # to annotate every member in every test, assert, etc, when we deprecate something)
     deprecated_member_use_from_same_package: warning
+  exclude:
+    - example/**
+    - example_web/**
 
 linter:
   rules:
-    - prefer_relative_imports
\ No newline at end of file
+    prefer_relative_imports: true
+    unnecessary_brace_in_string_interps: true
```

Comment:
@rxlabz if I got this right, the format should be `name: true or false`, so we preserve the values from pedantic.

https://dart.dev/guides/language/analysis-options#disabling-individual-rules

the way how it is, it basically ignores the pedantic file and only activates the `prefer_relative_imports` one which is not what we want right.

Reply:
I'm not sure, I think you only need the map syntax if you want to disable some rules

Reply:
> If you include an analysis options file such as the one in pedantic, you might want to disable some of the included rules. Disabling individual rules is similar to enabling them, but requires the use of a map rather than a list as the value for the rules

Reply:
that's pretty clear for me tbh, but not a `pedantic` expert here so :(

Reply:
> Disabling individual rules is similar to enabling them, but requires the use of a map

not that clear, but seems to me that only "disabling rules... requires the use of a map" :)

Reply:
( as changing the syntax don't show new warning or hint, I'm pretty sure that the pedantic rules were applied without the map syntax )

Reply:
tbh I'm not sure, `prefer_relative_imports` was enabled already and it didn't fail our build before that, see the `sentry_navigation_observer.dart ` file

Reply:
most of the rules only show hints and warnings, not errors. You need to add them to the analyzer:errors section if you want them to be considered as compilation errors.